rpcs3/Emu/Cell/lv2/sys_process.cpp

```diff
@@ -402,7 +402,7 @@ void _sys_process_exit2(ppu_thread& ppu, s32 status, vm::ptr<sys_exit2_param> ar
 
 	// TODO: set prio, flags
 
-	lv2_exitspawn(ppu, argv, envp, data);
+	return lv2_exitspawn(ppu, argv, envp, data);
```

Comment:
🤔

Reply:
Just like how` sys_process_exit3()` returns `_sys_process_exit()` and how `exitspawn()` (in sys_game_.cpp) returns `_sys_process_exit()` or `_sys_process_exit2()`, though I don't believe a void needs a return value (even returning a void).

Reply:
Uh it just a trick to shorten code, it doesn't mean anything and only causes confusion when not needed.